plugins/UM3NetworkPrinting/src/Cloud/CloudOutputController.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+from cura.PrinterOutput.PrinterOutputController import PrinterOutputController
+
+
+class CloudOutputController(PrinterOutputController):
+    def __init__(self, output_device):
```

Comment:
```suggestion
    def __init__(self, output_device: OutputDevice) -> None:
```
Don't forget to also add the import for the output device